eng/pipelines/common/variables.yml

```diff
@@ -34,4 +34,11 @@ variables:
   ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
     value: Debug
 
+- name: wasmDarcDepsChanged
+  value: $[ or(
+              eq(dependencies.evaluate_paths.outputs['DarcDependenciesChanged.Microsoft_NET_Workload_Emscripten_Manifest-7_0_100'], true),
```

Comment:
can we somehow do a string contains check here so we don't need to include the `7_0_100` version number? we'll almost certainly forget to update this here.

Reply:
yeah, I'll try